features/resources/v2algodclient_responsejsons/sourcemap.json

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["quine.teal"],"names":[],"mapping":";AAOA;;;;;;;;;;;;;;;;;;;;;;;;;;;;AACA;AACA;;;AACA;;;AACA;AACA;;AACA;AACA;;;AACA;AACA;AACA;;AACA;;AACA;AACA;AACA","mappings":";AAOA;;;;;;;;;;;;;;;;;;;;;;;;;;;;AACA;AACA;;;AACA;;;AACA;AACA;;AACA;AACA;;;AACA;AACA;AACA;;AACA;;AACA;AACA;AACA"}
```

Comment:
@barnjamin Asking without having research under the assumption that you know off-hand:  What requires duplicating `sourcemap.json` in 2 locations rather than retaining a single copy?

Reply:
I don't know in fact, mostly doing it out of superstition

Reply:
Gotcha - I imagine only 1 copy is needed.  I'd like to leave the thread open as a visual reminder to investigate.

Reply:
@barnjamin Based on a quick review of Python + Go SDKs, the resource loading method references `features/resources`.

The finding suggests to me that `features/unit/v2algodclient_responsejsons/sourcemap.json` can be removed.

If you agree with the rationale, can you remove `features/unit/v2algodclient_responsejsons/sourcemap.json` and confirm downstream builds pass?